dataprofiler/labelers/column_name_model.py

```diff
@@ -0,0 +1,248 @@
+"""Contains class for column name data labeling model."""
+import json
+import os
+
+import numpy as np
+
+try:
+    from rapidfuzz import fuzz, process
+except ImportError:
+    raise ImportError("Please ensure you run `pip install requirements-ml.txt`")
```

Comment:
this is pretty much guaranteed to run which means this would happen all the time for any model, even the regex one.
However, doing something like:
https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/reports/graphs.py#L12
with:
https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/reports/graphs.py#L21
would alllow modules to import, but only error on function / class calls